src/main/java/org/apache/sysml/api/mlcontext/MLContext.java

```diff
@@ -100,6 +100,11 @@
 	private boolean statistics = false;
 
 	/**
+	 * The number of heavy hitters that are printed as part of the statistics option
+	 */
+	private int statisticsHeavyHitters = 10;
```

Comment:
Hi @nakul02. Would you mind updating the field **statisticsHeavyHitters** to **statisticsMaxHeavyHitters** to match the setStatisticsMaxHeavyHitters setter name and statisticsMaxHeavyHitters in ScriptExecutor?